src/Commands/PerformanceTable.php

```diff
@@ -0,0 +1,84 @@
+<?php namespace CodeIgniter\Tasks\Commands;
+
+use CodeIgniter\CLI\BaseCommand;
+use CodeIgniter\CLI\GeneratorTrait;
+use CodeIgniter\Services;
+
+/**
+ * Generates a Tasks' Perforamance Table Migration file.
+ */
+class PerformanceTable extends TaskCommand
+{
+	use GeneratorTrait;
+
+	/**
+	 * The Command's Name
+	 *
+	 * @var string
+	 */
+	protected $name = 'tasks:performance-table';
+
+	/**
+	 * The Command's Description
+	 *
+	 * @var string
+	 */
+	protected $description = 'Generates a tasks\' performance migration file.';
+
+	/**
+	 * The Command's Usage
+	 *
+	 * @var string
+	 */
+	protected $usage = 'tasks:performance-table';
+
+	/**
+	 * Actually execute a command.
+	 *
+	 * @param array $params
+	 */
+	public function run(array $params)
+	{
+		$this->component = 'Migration';
+		$this->directory = 'Database\Migrations';
+		$this->template  = 'performance_table.tpl.php';
+
+		$params[0] = "_create_performance_tasks_table";
+
+		$this->execute($params);
+	}
+
+
+	/**
+	 * Change file basename before saving.
+	 *
+	 * @param string $filename
+	 *
+	 * @return string
+	 */
+	protected function basename($filename): string
+	{
+		// Set a unqiue name to avoid writing more than one migration file
+		return "2021-01-01-000000_" . basename($filename);
```

Comment:
Commented line 61 explaining why we override the basename function.
If you meant this commented line on line 63, It was deleted on purpose and not only for testing. to avoid creating multiple migrations files for the same table.
```php
//return gmdate(config('Migrations')->timestampFormat) . basename($filename);
```

I'm using the same command structure in my projects for my CI4 packages to create unique migrations.